connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -106,6 +108,17 @@ public void stop() {
         Utils.closeQuietly(targetAdminClient, "target admin client");
     }
 
+    @Override
+    public Config validate(Map<String, String> connectorConfigs) {
+        List<ConfigValue> configValues = super.validate(connectorConfigs).configValues();
+        new MirrorCheckpointConfig(connectorConfigs).validate().forEach(invalidConfig ->
+                configValues.stream()
+                        .filter(conf -> conf.name().equals(invalidConfig.name()))
+                        .forEach(conf -> invalidConfig.errorMessages().forEach(msg -> conf.addErrorMessage(msg))));
```

Comment:
Holy Java 8 Batman!

We don't need `forEach` here, can simplify a bit:
```suggestion
                        .forEach(conf -> conf.errorMessages().addAll(invalidConfig.errorMessages())));
```

Reply:
changed this now